grouper/fe/forms.py

```diff
@@ -18,6 +18,12 @@
 from .. import model_soup
 
 
+GROUP_CANJOIN_CHOICES = [
+    ("canjoin", "Anyone"), ("canask", "Must Ask"),
+    ("nobody", "Nobody"),
+    ]
```

Comment:
style nit: the closing `]` should be unindented. there's some inconsistency in this file already, so I'll just clean it all up later.